package.json

```diff
@@ -98,5 +100,8 @@
   },
   "volta": {
     "node": "16.7.0"
+  },
+  "devDependencies": {
```

Comment:
devDependencies don't make sense for roots in monorepos, you can move this into `dependencies`.